### DISCUSSION TITLE
Announcement on Eng-Tips.com

### DISCUSSION THREAD

**Discussion by Bob-May** [mgreminger/EngineeringPaper.xyz#189] [General]
I noticed the other day that you posted an update announcement on Eng-TIps.com, following up on a prior announcement in July of 2021.

Both of those announcements have been deleted by Eng-Tips management.

On Eng-Tips "I'm an Eng-Tips Forums Fellow and member of the Eng-Tips Forums Round Table, where management is advised on site operations and proposed programming enhancements."

To be clear, this gives me exactly zero authority at Eng-Tips.  I am not an Eng-Tips employee and do not speak for Eng-Tips in any way.

Regardless, your posts did not appear to me to violate any Eng-Tips policy, and it was not clear to me why your announcements were removed.  Other members of the Eng-Tips Round Table agree with my opinion.

I have asked Eng-Tips management why your announcements were removed.  You might also ask yourself at [info@engineering.com](mailto:info@engineering.com)

https://simpliengineering.com/ was created by a small group of Eng-Tips Round Table members in response to dissatisfaction with a change of ownership at Eng-Tips some years ago.  An announcement of EngineeringPaper.xyz would be welcome there.

**Comment by mgreminger** [comment]
Thanks for message and the additional background information, much appreciated. Yes, they were pretty quick to delete the post. They actually deleted the 2021 post as well and deleted my account, all without any notice or explanation. I figured it was management since my experience with the eng-tips community has been very positive. With my post the other day, I has updating the community on what has changed since the original announcement, a lot of it was suggested by the eng-tips community in that original, now deleted, post.

Thanks for letting me know about simpliengineering.com, I'll definitely check it out!
